test/pnts_parser_unit_test.js

```diff
@@ -0,0 +1,46 @@
+import assert from 'assert';
+import PntsParser from '../src/Parser/PntsParser';
+
+function bufferFromString(pnts, size) {
+    const buffer = new ArrayBuffer(size);
+    const typed = new Uint8Array(buffer);
+    let next = 0;
+    for (const line of pnts.split('\n')) {
+        for (const word of line.split(' ')) {
+            if (word.length == 2) {
+                typed[next++] = parseInt(word, 16);
+            }
+        }
+    }
+    assert.equal(next, size);
+    return buffer;
+}
+
+describe('pnts parser', function () {
+    const pnts = `
```

Comment:
Not a big fan of this, I saw some other methods (like base64 string) but all seem as no looking this good.